api_tester/lib/models/customer_info_wrapper_api_test.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:purchases_flutter/object_wrappers.dart';
+
+// ignore_for_file: unused_element
+// ignore_for_file: unused_local_variable
+class _CustomerInfoApiTest {
+  void _checkFromJsonFactory(Map<String, dynamic> json) {
+    CustomerInfo customerInfo = CustomerInfo.fromJson(json);
+  }
+
+  void _checkToJson(CustomerInfo customerInfo) {
+    Map<String, dynamic> json = customerInfo.toJson();
+  }
+
+  void _checkConstructor(
```

Comment:
Same as with the `fromJson` and `toJson` methods, don't think we want the constructor to be public, but it currently is so I added api tests for them.